src/test/java/org/mozilla/taskcluster/urls/URLsTest.java

```diff
@@ -60,10 +60,18 @@ public static String testFunc(String function, URLProvider urlProvider, String[]
             return urlProvider.exchangeReference(args[0], args[1]);
         case "schema":
             return urlProvider.schema(args[0], args[1]);
+        case "apiReferenceSchema":
+            return urlProvider.apiReferenceSchema(args[0]);
+        case "exchangesReferenceSchema":
+            return urlProvider.exchangesReferenceSchema(args[0]);
+        case "apiManifestSchema":
+            return urlProvider.apiManifestSchema(args[0]);
+        case "metadataMetaschema":
+            return urlProvider.metadataMetaschema();
```

Comment:
your java build is failing because of this line

Reply:
needs a string passed in

Reply:
Thanks, silly me.  Fix pushed.